unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
This is definitely true for the official Window build system (GNU Makefile). This is apparently true for MSVC2013 build system, or how it can build stuff at all?

But the main question, how is that possible that MSVC2013 (and any other compiler) doesn't need "settimeout" qstr, and MSVC2015 needs it?